public/app/percona/shared/components/PerconaBootstrapper/PerconaUpdateVersion/PerconaUpdateVersion.tsx

```diff
@@ -59,13 +59,13 @@ const PerconaUpdateVersion = () => {
       <Modal
         onDismiss={onDismiss}
         title={Messages.titleOneUpdate}
-        isOpen={showUpdateModal && changeLogs && changeLogs?.updates?.length === 1}
+        isOpen={showUpdateModal&&( !!!changeLogs|| (changeLogs && changeLogs?.updates?.length <= 1))}
```

Comment:
I think the check for changelogs can be simplified a bit, something like `(changeLogs?.updates?.length || 0) <= 1 ` should be enough.